packages/react-native/Libraries/Utilities/DevMenu.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import NativeDevMenu from '../NativeModules/specs/NativeDevMenu';
+
+let DevMenu: {
+  show(): void,
+} = {
+  show(): void {},
+};
+
+if (__DEV__) {
+  DevMenu = {
+    show(): void {
+      NativeDevMenu.show?.();
+    },
+  };
+}
```

Comment:
Can be simplified by moving `if (__DEV__)` inside, which is compiled out equivalently.

```suggestion
/**
 * The DevMenu module exposes methods for interacting with the Dev Menu in development.
 */
type DevMenuStatic = {
  /**
   * Show the Dev Menu.
   */
  show(): void,
};

const DevMenu: DevMenuStatic = {
  show(): void {
    if (__DEV__) {
      NativeDevMenu.show?.();
    }
  },
};
```